examples/thinking-react/src/components/ProductRow/ProductRow.js

```diff
@@ -0,0 +1,16 @@
+const ProductRow = ({ product }) => {
```

Comment:
Only pass what you need instead of pass the whole `product` here

```ts
const ProductRow = ({ name, price }) =>
```

Reply:
fixed